### PR TITLE
fix: enabling exit codes for general cli o/p and updating field names

### PR DESCRIPTION
**Pull request by deepak1725**
Enabling exit codes for general CLI output and updating field names

Jira: https://issues.redhat.com/browse/APPAI-1816

### CODE REVIEW

**Review by yzainee — CHANGES_REQUESTED** [review]
Please also fix the spelling in
https://github.com/fabric8-analytics/cli-tools/blob/5254513a6a30ff054e9adced69755ec5613ad497/analyses/summary/types.go#L15 & 16
Change needed: vulnerabilites -> vulnerabilities

File: analyses/verbose/testdata/verbosedata.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 				"version": "1.1"
 			}],
 			"latest_version": "7.1.2",
-			"commonly_known_vulnerabilities": [
+			"publicly_available_vulnerabilities": [
```
Comment (arajkumar) [inline analyses/verbose/testdata/verbosedata.json:10]: What is the reason for changing the text. I guess we use `commonly_known` to denote public vulnerabilities across the board.

Comment (yzainee) [reply analyses/verbose/testdata/verbosedata.json:10]: @arajkumar it was a comment from krishna to change this text to publicly available vuln

Comment (deepak1725) [reply analyses/verbose/testdata/verbosedata.json:10]: @arajkumar https://issues.redhat.com/browse/APPAI-1816

**Review by yzainee — DISMISSED** [review]
LGTM

File: analyses/verbose/types.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ type DependenciesType struct {
 	Transitives                  []DependenciesType  `json:"transitives"`
 	LatestVersion                string              `json:"latest_version"`
 	RecommendedVersion           string              `json:"recommended_version"`
-	CommonlyKnownVulnerabilities []VulnerabilityType `json:"commonly_known_vulnerabilities"`
+	CommonlyKnownVulnerabilities []VulnerabilityType `json:"publicly_available_vulnerabilities"`
```
Comment (arajkumar) [inline analyses/verbose/types.go:32]: Should we change the field name as well? @yzainee ?
CommonlyKnownVulnerabilities => PubliclyAvailableVulnerabilities?

Comment (yzainee) [reply analyses/verbose/types.go:32]: it wouldnt matter much in the end. will leave it to @deepak1725

Comment (deepak1725) [reply analyses/verbose/types.go:32]: @arajkumar its internal usage only.. I feel it's okay to keep it as it is. 
Please let me know if you have strong opinions regarding that.

Comment (deepak1725) [reply analyses/verbose/types.go]: Anyway, included that update also
 > CommonlyKnownVulnerabilities => PubliclyAvailableVulnerabilities?

**Review by yzainee — APPROVED** [review]

**Pull request merged**